docs/website/netcdf-java/reference/ToolsUI/ToolsUI.adoc

```diff
@@ -66,7 +66,7 @@ Note: to write to NetCDF-4, you must have the link:../netcdf4Clibrary.html[C lib
 image:writer1.png[Writer tab]
 
 === NCDump
-The NCDump tab offers functionality similar to�albeit more limited than�the
+The NCDump tab offers functionality similar to�albeit more limited than�the
```

Comment:
Two more and that should have it :-)

Reply:
Lame.  Sorry I missed that.  It's been corrected.

Reply:
@lesserwhirls. Ok, 'tis done.